packages/appsync-modelgen-plugin/package.json

```diff
@@ -40,12 +40,13 @@
   },
   "devDependencies": {
     "@graphql-codegen/testing": "^1.17.7",
-    "graphql": "^14.5.8",
+    "@graphql-codegen/typescript": "^2.8.3",
```

Comment:
Test would fail with the following without including this.
```
FAIL src/__tests__/visitors/appsync-javascript-visitor.test.ts
  ● Test suite failed to run

    Cannot find module '@graphql-codegen/typescript' from '../../node_modules/@graphql-codegen/testing/cjs/resolvers-common.js'
```

I think GraphQL 14 may have included this dependency before.